views/ui/toolbox.php

```diff
@@ -63,7 +63,7 @@ class="wu-inline-block wu-uppercase wu-text-gray-600 wu-no-underline">
 				class="dashicons-wu-circular-graph wu-text-sm wu-w-auto wu-h-auto wu-align-text-bottom wu-relative"></span>
 			<span class="">
 				<?php // translators: %s hash of membership. ?>
-				<?php printf(wp_kses_post('Membership <Strong>%s</strong>', 'ultimate-multisite'), esc_html($membership->get_hash())); ?>
+				<?php printf(wp_kses_post(sprintf(__('Membership <Strong>%s</strong>', 'ultimate-multisite'), esc_html($membership->get_hash())))); ?>
```

Comment:
```suggestion
				<?php printf(wp_kses_post(__('Membership <Strong>%s</strong>', 'ultimate-multisite')), esc_html($membership->get_hash())); ?>
```

Reply:
Thanks for submitting this PR. The __ must of gotten lost during some refactor. Your code works but it's rather redundant to use both printf and sprintf.